### DISCUSSION TITLE
Layout v2 unavailable

### DISCUSSION THREAD

**Discussion by steakhutzeee** [firefly-iii/firefly-iii#5685] [Firefly III] [answered]
Hi,

sorry if this has been already discussed. I read in latest version the banner stating that layout v2 would not be updated anymore.

I just updated and found i'm on layout v1. So layout v2 has been completely removed? I quite do not remember if this was stated in the banner of previous versions.

If this is so, how should we edit the `FIREFLY_III_LAYOUT=v2` entry?

Thanks!

**Comment by steakhutzeee** [comment]
Bump.

**Comment by JC5** [comment, marked as answer]
v2 has been removed, sorry about that. For now, the var does nothing.

**Reply by Extarys** [reply]
Oh that explains it... Thanks! I tried too for an hour without success :laughing:
